README.md

```diff
@@ -53,56 +77,65 @@ transport protocols:
 
 RTPS (Interoperability) features are based on the [DDS-RTPS Specification
 formal/2014-09-01 (version 2.2)](http://www.omg.org/spec/DDSI-RTPS/2.2).  See
-the OpenDDS Developer's Guide and the file docs/design/RTPS for more details on
-RTPS.
+the OpenDDS Developer's Guide and the file [docs/design/RTPS](docs/design/RTPS)
+for more details on RTPS.
 
 See the [Developer's Guide](http://download.objectcomputing.com/OpenDDS) for
 information on OpenDDS compliance with the DDS specification. If you would like
 to contribute a feature or sponsor the developers to add a feature  please see
 the Support section above for contact information.
 
 
-# Dependencies
+## Dependencies
 
-## TAO
+For a complete detailed list of dependencies, see
+[docs/dependencies.md](docs/dependencies.md).
+
+### TAO
 
 OpenDDS requires TAO for both IDL compilation as well as interaction
 with the DCPSInfoRepo.  If you will be using the "configure" script for OpenDDS
-(see the INSTALL file for details), you do not need to download TAO first --
+(see the [INSTALL.md](INSTALL.md) file for details), you do not need to download TAO first --
 the "configure" script will download it for you.
 
 Use one of the following versions when building OpenDDS:
 
-* TAO 2.2a patch 15
-* TAO 2.5.1 (DOC Group)
-
-Note that the 2.2a release is from OCI and can be obtained
-from http://www.theaceorb.com/.  The DOC Group releases can be obtained from
-http://download.dre.vanderbilt.edu/.
-
-## GNU Make
+* OCI TAO 2.2a patch 15
+  * Available from:
+    * http://download.objectcomputing.com/TAO-2.2a_patches/
+  * This version will be automatically downloaded by default when using the
+    configure script.
+* DOC Group TAO 2.5.1
+  * Available from:
+    * [github.com/DOCGroup/ACE_TAO](https://github.com/DOCGroup/ACE_TAO/releases)
+    * http://download.dre.vanderbilt.edu/
+  * When using the configure script, this version can be downloaded using one
```

Comment:
Do you want me to use the word "distribution" instead of "version"? Then I'm not sure what you mean in the second sentence, do you want spell out that `--doc_group` is stable and `--ace-github-latest` isn't?

So maybe something like this:

* DOC Group TAO 2.5.1
  * Available from:
    * [github.com/DOCGroup/ACE_TAO](https://github.com/DOCGroup/ACE_TAO/releases)
    * http://download.dre.vanderbilt.edu/
  * When using the configure script, this distribution can be downloaded using one
    of these arguments:
    * `--doc_group` for the latest tested release
    * `--ace-github-latest` to use the current untested master branches of ACE_TAO and MPC